src/AccessibilityInsights.Extensions/Container.cs

```diff
@@ -107,8 +106,8 @@ private Container()
         [Import(typeof(ITelemetry), AllowDefault = true)]
         public ITelemetry Telemetry { get; set; }
 
-        [Import(typeof(IBugReporting), AllowDefault = true)]
-        public IBugReporting BugReporting { get; set; }
+        [ImportMany(typeof(IIssueReporting))]
+        public IEnumerable<IIssueReporting> IssueReporting { get; set; }
```

Comment:
since this is enumerable now, would IssueReportings or IssueReportingMethods be better? (plural)

Reply:
Agreed. Went with IssueReportingOptions.